packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -43,91 +46,98 @@ function BlockPatternsSearchResults( { filterValue, onInsert } ) {
 	}
 }
 
-function BlockPatternsPerCategories( { onInsert } ) {
-	const [ patterns, categories, onClick ] = usePatternsState( onInsert );
+function BlockPatternsCategory( {
+	selectedCategory,
+	onClickCategory,
+	onInsert,
+} ) {
+	const [ allPatterns, allCategories, onClick ] = usePatternsState(
+		onInsert,
+		selectedCategory
+	);
+
+	const [ currentCategorypatterns, setCategoryPatterns ] = useState( [] );
```

Comment:
`currentCategorypatterns` should be `currentCategoryPatterns`.

Reply:
Doh! thanks, fixed.